PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/forms/AddNewClusterForm.java

```diff
@@ -65,6 +75,19 @@ protected void configureShell(Shell newShell) {
         newShell.setText("Link New HDInsight Cluster");
     }
 
+    private void refreshContainers(@NotNull ClientStorageAccount storageAccount) {
+    	try {
+        	containersComboBox.removeAll();
+
+			StorageClientSDKManager.getManager().getBlobContainers(storageAccount.getConnectionString())
+					.forEach(blob -> containersComboBox.add(blob.getName()));
+			
+			// not find setMaximumRowCount(*) method in SWT
+		} catch (AzureCmdException e) {
+			containersComboBox.removeAll();
```

Comment:
>containersComboBox.removeAll(); [](start = 3, length = 31)

Probably we should not removeAll? What's wrong with the containers that are already fetched.

Reply:
Recommend to delete "removeAll" and add error message here

---
In reply to: [164978754](https://github.com/Microsoft/azure-tools-for-java/pull/1360#discussion_r164978754) [](ancestors = 164978754)

Reply:
"removeAll" is deleted and error message is added.